schemahandle_operations.py

```diff
@@ -0,0 +1,89 @@
+from space_etl import PostgresqlDestination
+
+
+class SchemaDriftHandle(PostgresqlDestination):
+    def __init__(self, db_name) -> None:
+        super().__init__(db_name)
+    
+    def select_existing_columns(self,details):
+        db_name = 'my_database1'
```

Comment:
`db_name` should come from Parent Class instead of putting manual value as a variable